Library/Homebrew/test/bottle_publisher_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "bottle_publisher"
+
+describe BottlePublisher do
+  let(:tap) { CoreTap.new }
+
+  changed_formulae_names = ["#{CoreTap.instance.name}/hello.rb"]
+  bottle_publisher = described_class.new(
+    CoreTap.instance, changed_formulae_names, "homebrew", false, false
+  )
+
+  describe "publish_and_check_bottles" do
+    it "fails if ENV[\"HOMEBREW_DISABLE_LOAD_FORMULA\"] is set to 1" do
```

Comment:
Use `'` instead of `"` rather than escaping `"`

Reply:
I will change this to: `it "fails if HOMEBREW_DISABLE_LOAD_FORMULA is set to 1" do`